src/v/pandaproxy/schema_registry/util.h

```diff
@@ -32,6 +32,8 @@ namespace pandaproxy::schema_registry {
 template<typename Encoding>
 result<schema_definition> make_schema_definition(std::string_view sv) {
     // Validate and minify
+    // TODO (Ben): Minify. e.g.:
```

Comment:
should we just create a github tix here ?

Reply:
I'll probably get to this in a couple of days with compatibility checks.